Instant action for setting certificates (#372)
VDA5050/VDA5050: VDA5050_EN.md

waitForTrigger | - | AGV has to wait for a trigger on the AGV (e.g., button press, manual loading). <br>Master control is responsible to handle the timeout and has to cancel the order if necessary. | yes | triggerType(string) | - | no | yes | no
cancelOrder | - | AGV stops as soon as possible. <br>This could be immediately or on the next node. <br>Then the order is deleted. All actions are canceled. | yes | - | - | yes | no | no
factsheetRequest | - | Requests the AGV to send a factsheet | yes | - | - | yes | no | no

> setCertificates | - | Request the AGV to download and activate a new certificate set | yes | service (string)<br>caDownloadLink (string)<br>keyDownloadLink (string)<br>certDownloadLink (string, optional) | - | yes | no | no |

Reviewer: ```suggestion
updateCertificate | - | Request the mobile robot to download and activate a new certificate set, the service parameter is an extendable enum with the predefined parameter 'MQTT' to be used for mqtt connection.  | yes | service (string)<br>caDownloadLink (string, optional)<br>keyDownloadLink (string)<br>certDownloadLink (string) | - | yes | no | no |
```

waitForTrigger | - | AGV is waiting for the trigger | - | Trigger has been triggered. | waitForTrigger fails, if order has been canceled.
cancelOrder | - | AGV is stopping or driving, until it reaches the next node. | - | AGV stands still and has canceled the order. | -
factsheetRequest | - | - | - | The factsheet has been communicated | -

> setCertificates | - | Vehicle is downloading and installing certificates | - | Certificates are downloaded, installed and active. | Download or installation failed. |

Reviewer: ```suggestion
updateCertificate | - | Vehicle is downloading and installing certificates | - | Certificates are downloaded, installed and active. | Download or installation failed. |
```

> ### 6.10.3 Set vehicle certificates

Reviewer: ```suggestion
### 6.10.3 Update vehicle certificate
```

> For security reasons, vehicle communication (at least for fleet management) should be secured. Typically, communication to the MQTT broker is secured via TLS, which requires one or more root certificates and a vehicle-specific key pair. The parameter `service` specifies the service (e.g. MQTT) for which the certificates are to be used. The parameter `caDownloadLink` specifies the URL for the root certificate(s). The parameters `certDownloadLink` and `keyDownloadLink` specify the URLs for the vehicle-specific public and private keys.

Reviewer: ```suggestion
For security reasons, vehicle communication (at least for fleet management) should be secured. Typically, communication to the MQTT broker is secured via TLS, which requires one or more root certificates and a vehicle-specific key pair. The parameter `service` specifies the service (e.g., 'MQTT') for which the certificates are to be used. The parameter `caDownloadLink` specifies the URL for the root certificate(s). The parameters `certDownloadLink` and `keyDownloadLink` specify the URLs for the vehicle-specific public and private keys.
```
